WordPress/Classes/Services/MenusService.m

```diff
@@ -9,8 +9,6 @@
 #import "WordPress-Swift.h"
 @import WordPressKit;
 
-NS_ASSUME_NONNULL_BEGIN
```

Comment:
Why was this removed? I'm guessing because it's not necessary in `.m` files?

Reply:
Indeed.  No benefit whatsoever in using it in .m files.